deploy/helm_charts/envs/mixer_autopush.yaml

```diff
@@ -34,6 +34,17 @@ serviceGroups:
       resources:
         memoryRequest: "400M"
         memoryLimit: "400M"
+  - mixer-default-service:
```

Comment:
mixer-observations-service?

Reply:
updated all the name to use just the label.